README.rst

```diff
@@ -1851,6 +1852,12 @@ Choose from the list of available rules:
 
   Unary operators should be placed adjacent to their operands.
 
+  Configuration options:
+
+  - ``not_operator_space`` (``'leading_and_trailing'``, ``'no_trailing'``,
```

Comment:
This got me confused as the new user of PHP CS Fixer - why only those 3 options? Where are `no_leading_and_trailing`, `no_leading` and `one_leading` options? Are they "worse" then added 3?

Wouldn't be more clear to have 2 boolean options `leading` and `trailing` with value `true|false|null` to cover all combination?

Reply:
I moved the current logic of the fixers into one and made the difference in behavior available through these configuration options. I didn't spend time on investigating new or better options, so thanks for the suggestion!

Reply:
Ok, and `no_trailing` is the default for new fixer, I missed that, now I get it 👍